oat-sa/extension-tao-testqti-previewer
# Fix/con 24/top block styles

Related to: https://oat-sa.atlassian.net/browse/CON-24

Added a border: solid 1 px #E5E4E4.
Switched the cursor to a pointer while it hovers the collapser: class="top-block-preview-collapser"

How to test:
- go to Tests page
- go to Test Preview
- check border and cursor on the bottom part
![image](https://user-images.githubusercontent.com/25976342/97958449-40051a00-1dbe-11eb-8f30-7b22facb1525.png)
